### PR TITLE
etcd updates 

### PR DESCRIPTION
**Pull request by noramullen1**
Addresses #1414

### CODE REVIEW

File: content/chronograf/v1.8/administration/create-high-availability.md [hunk 1]
```diff
@@ -26,16 +26,36 @@ Have an existing Chronograf configuration store that you want to use with a Chro
 2. Extract the `etcd` binary and place it in your system PATH.
 3. Start etcd.
 
+
 ## Start Chronograf
```
Comment (timhallinflux) [inline content/chronograf/v1.8/administration/create-high-availability.md:30]: super clear!

File: content/chronograf/v1.8/tools/chronograf-cli/_index.md [hunk 2]
```diff
@@ -84,15 +84,15 @@ chronograf [flags]
 |:-------------------------------|:------------------------------------------------------------------------|:--------------------|
 | `-i`, `--github-client-id`     | GitHub client ID value for OAuth 2.0 support                            | `$GH_CLIENT_ID`     |
 | `-s`, `--github-client-secret` | GitHub client secret value for OAuth 2.0 support                        | `$GH_CLIENT_SECRET` |
-| `-o`, `--github-organization`  | Specify a GitHub organization membership required for a user. Optional. | `$GH_ORGS`          |
+| `-o`, `--github-organization`  | Restricts authorization to users from comma-separated list of specified Github organizations. Optional. | `$GH_ORGS`          |
```
Comment (timhallinflux) [inline content/chronograf/v1.8/tools/chronograf-cli/_index.md:87]: I don't think this matches what the underlying issue outlines.  These are flags... and when using flags it implies having multiple flags.  

When using environment variables --> then the comma-separated list works.

File: content/chronograf/v1.8/tools/chronograf-cli/_index.md [hunk 3]
```diff
@@ -84,15 +84,15 @@ chronograf [flags]
...
 
 ### Google-specific OAuth 2.0 authentication flags
 
 | Flag                     | Description                                                                     | Env. Variable           |
 |:-------------------------|:--------------------------------------------------------------------------------|:------------------------|
 | `--google-client-id`     | Google client ID value for OAuth 2.0 support                                    | `$GOOGLE_CLIENT_ID`     |
 | `--google-client-secret` | Google client secret value for OAuth 2.0 support                                | `$GOOGLE_CLIENT_SECRET` |
-| `--google-domains`       | Restricts authorization to users from specified Google email domains. Optional. | `$GOOGLE_DOMAINS`       |
+| `--google-domains`       | Restricts authorization to users from comma-separated list of specified Google email domains. Optional. | `$GOOGLE_DOMAINS`       |
```
Comment (timhallinflux) [inline content/chronograf/v1.8/tools/chronograf-cli/_index.md:95]: I don't think this matches what the underlying issue outlines.  These are flags... and when using flags it implies having multiple flags.  

When using environment variables --> then the comma-separated list works.

File: content/chronograf/v1.8/tools/chronograf-cli/_index.md [hunk 4]
```diff
@@ -102,15 +102,15 @@ chronograf [flags]
...
 | `--auth0-client-id`     | Auth0 client ID value for OAuth 2.0 support                                                                                                                                                                                   | `$AUTH0_CLIENT_ID`     |
 | `--auth0-client-secret` | Auth0 client secret value for OAuth 2.0 support                                                                                                                                                                               | `$AUTH0_CLIENT_SECRET` |
-| `--auth0-organizations` | Auth0 organization membership required to access Chronograf. Organizations are set using an organization key in the user’s `app_metadata`. Lists are comma-separated and are only available when using environment variables. | `$AUTH0_ORGS`          |
+| `--auth0-organizations` | Restricts authorization to  users from comma-separated list of specified Auth0 organizations. Optional. Organizations are set using an organization key in the user’s `app_metadata`.  | `$AUTH0_ORGS`          |
```
Comment (timhallinflux) [inline content/chronograf/v1.8/tools/chronograf-cli/_index.md:105]: I don't think this matches what the underlying issue outlines.  These are flags... and when using flags it implies having multiple flags.  

When using environment variables --> then the comma-separated list works.

File: content/chronograf/v1.8/tools/chronograf-cli/_index.md [hunk 5]
```diff
@@ -102,15 +102,15 @@ chronograf [flags]
...
 
 ### Heroku-specific OAuth 2.0 authentication flags
 
 | Flag                    | Description                                                                              | Env. Variable       |
 |:------------------------|:-----------------------------------------------------------------------------------------|:--------------------|
 | `--heroku-client-id`    | Heroku client ID value for OAuth 2.0 support                                             | `$HEROKU_CLIENT_ID` |
 | `--heroku-secret`       | Heroku secret for OAuth 2.0 support                                                      | `$HEROKU_SECRET`    |
-| `--heroku-organization` | Heroku organization membership required to access Chronograf. Lists are comma-separated. | `$HEROKU_ORGS`      |
+| `--heroku-organization` | Restricts authorization to  users from comma-separated list of specified Heroku orgnizations. Optional. | `$HEROKU_ORGS`      |
```
Comment (timhallinflux) [inline content/chronograf/v1.8/tools/chronograf-cli/_index.md:113]: I don't think this matches what the underlying issue outlines.  These are flags... and when using flags it implies having multiple flags.  

When using environment variables --> then the comma-separated list works.

**Review by timhallinflux — CHANGES_REQUESTED** [review]
if we have multiple entries for a particular param, multiple command line flags are supported.
unless you are using environment variables....in which case the comma separated list is the way.

Need to confirm for each of the items listed in the underlying issue are covered (either in this PR or another?).

**Review by timhallinflux — CHANGES_REQUESTED** [review]
where do we specify how to handle the comma separated list when using an Env Var?

I think we should add this content here: https://docs.influxdata.com/chronograf/v1.8/administration/config-options/

under each env var where it applies?

We've got some of this already....but the language is inconsistent: ex -> Lists are comma-separated and are only available when using environment variables.

Instead of saying "lists" -- I would suggest something like:  multiple <thing> can be specified using a comma-separated list. Then show an example.

**Review by timhallinflux — CHANGES_REQUESTED** [review]
looks great.  Seems like there is one example still missing:
`--heroku-organization=`

I think we hit that one and this is ready to go!

**Review by timhallinflux — APPROVED** [review]
gtg!

**Review by kelseiv — APPROVED** [review]

**Pull request merged**